### PR TITLE
Add Containerfile location e2e test

### PR DESCRIPTION
**Pull request by saschagrunert**
As a follow up of the location fix in #5080 we now add an e2e test for
that use case.

### CODE REVIEW

**Comment by rhatdan** [comment]
/approve
lgtm

**Comment by TomSweeneyRedHat** [comment]
LGTM
One test failed with an apparent network hiccup, I've restarted.

File: test/e2e/build_test.go [hunk 1]
```diff
@@ -105,4 +107,39 @@ var _ = Describe("Podman build", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 	})
+
+	It("podman build Containerfile locations", func() {
+		// Given
+		// Switch to temp dir and restore it afterwards
+		cwd, err := os.Getwd()
+		Expect(err).To(BeNil())
+		Expect(os.Chdir(os.TempDir())).To(BeNil())
```
Comment (mheon) [inline test/e2e/build_test.go]: @baude You're more familiar with the tests than I - would we expect this to cause any issues with parallel testing? I don't think so...

**Pull request merged**